server/views/components/captioned-image/index.njk

```diff
@@ -2,9 +2,9 @@
   <div class="captioned-image__image-container">
   {% set sizes = image | getImageSizesFor %}
   {% set comma = joiner() %}
-  <img class="image"
-        src="{{ image.contentUrl }}?w={{ sizes[3] }}"
-        srcset="
+  <img class="image lazyload"
```

Comment:
Do we maybe want to make sure we always load in the first image without JS?
Are we going to have links to the other perhaps for non-JS users?